### ISSUE TITLE
Bug report

### ISSUE THREAD

**Issue by Dmytro-Komaryst** [postmanlabs/newman#669]
Reproduced this "Bug" on the platform:
OS: Windows 7 and Windows 8.1
Windows app: Postman v4.6.1 and Postman v4.6.2
Newman v2 and v3
NodeJS v.4.5.1 and v.6.4.0

Received error for POST request (must be response "204 No Content") but I received next error:
```
TypeError: Expected a finite number
at module.exports (C:\Program Files\nodejs\node_modules\npm\node_modules\pretty-ms\index.js:8:9)
at C:\Program Files\nodejs\node_modules\npm\node_modules\newman\lib\reporters\html\index.js:75:27
at arrayEach (C:\Program Files\nodejs\node_modules\npm\node_modules\lodash\lodash.js:522:11)
at Function.forEach (C:\Program Files\nodejs\node_modules\npm\node_modules\lodash\lodash.js:9011:14)
at EventEmitter. (C:\Program Files\nodejs\node_modules\npm\node_modules\newman\lib\reporters\html\index.js:67:11)
at EventEmitter.emit (C:\Program Files\nodejs\node_modules\npm\node_modules\eventemitter3\index.js:138:33)
at .assignIn.done (C:\Program Files\nodejs\node_modules\npm\node_modules\newman\lib\run\index.js:156:29)
at C:\Program Files\nodejs\node_modules\npm\node_modules\postman-runtime\lib\backpack\index.js:56:34
at PostmanCollectionRun..extend._process (C:\Program Files\nodejs\node_modules\npm\node_modules\postman-runtime\lib\runner\run.js:107:20)
at PostmanCollectionRun. (C:\Program Files\nodejs\node_modules\npm\node_modules\postman-runtime\lib\runner\run.js:118:18)
```

**Comment by Dmytro-Komaryst** [comment]
This is a problem in postman-runtime, not this module pretty-ms.

**Comment by shamasis** [comment]
@Dmytro-Komaryst - irrespective of whose module is being used, it is our responsibility to fix it since we are the ones using it. 😉 

We will fix this asap.

**Comment by kunagpal** [comment]
@Dmytro-Komaryst This bug has been fixed in #671, and will be rolled out in the next Newman release.
